framework-plugins/lisk-framework-chain-connector-plugin/README.md

```diff
@@ -0,0 +1,78 @@
+# @liskhq/lisk-framework-chain-connector-plugin
+
+@liskhq/lisk-framework-chain-connector-plugin is a plugin for creating and sending Cross-Chain-Update (CCU) Transactions.
+
+Cross-chain update transactions are the carriers of the information transmitted between chains. By posting a cross-chain update, the receiving chain gets the information required about the advancement of the sending chain. The transaction can also include cross-chain messages and thus serves as an envelope for messages from one chain to another.
+
+## Installation
+
+```sh
+$ npm install --save @liskhq/lisk-framework-chain-connector-plugin
+```
+
+## Config Options
+
+```
+{
+	receivingChainID: string,
+	receivingChainWsURL?: string,
+	receivingChainIPCPath?: string,
+	ccuFrequency: number,
+	encryptedPrivateKey: string,
+	ccuFee: string,
+	isSaveCCU: boolean,
+	maxCCUSize: number,
+	registrationHeight: number,
+	ccuSaveLimit: number
+}
+```
+
+## Parameters
+
+| Param                   | Required? | Description                                                                    |
+| ----------------------- | --------- | ------------------------------------------------------------------------------ |
+| `receivingChainID`      | **Y**     | Chain ID of the receiving chain                                                |
+| `receivingChainWsURL`   | **N**     | The WS url of a receiving node                                                 |
+| `receivingChainIPCPath` | **N**     | The IPC path of a receiving node                                               |
+| `ccuFrequency`          | **Y**     | Number of blocks after which a CCU should be created                           |
+| `encryptedPrivateKey`   | **Y**     | Encrypted privateKey of the relayer                                            |
+| `ccuFee`                | **Y**     | Fee to be paid for each CCU transaction                                        |
+| `isSaveCCU`             | **Y**     | Flag for the user to either save or send a CCU on creation. Send is by default |
+| `maxCCUSize`            | **Y**     | Maximum size of CCU to be allowed                                              |
+| `registrationHeight`    | **Y**     | Height at the time of registration on the receiving chain                      |
+| `ccuSaveLimit`          | **Y**     | Number of CCUs to save                                                         |
```

Comment:
Just a small point but their are some default values for required properties so even if user skips those properties in the config, it will still work with default values